Packages/com.unity.inputsystem/InputSystem/Devices/Gamepad.cs

```diff
@@ -174,7 +171,7 @@ public struct GamepadState : IInputStateTypeInfo
         /// <summary>
         /// State format tag for GamepadState.
         /// </summary>
-        /// <value>Returns "GPAD".</value>
+        /// <remarks>Returns "GPAD".</remarks>
```

Comment:
I would guess this violates length requirement, is a remark required here? Could it be "State format tag for GamepadState ("GPAD")."

Reply:
Fixed